.github/workflows/verify-examples-sync.yml

```
@@ -0,0 +1,116 @@
+        with:
+          node-version-file: package.json
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build affected packages
+        run: pnpm build --filter=@baseplate-dev/project-builder-cli
+
+      - name: Run sync diff for example
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2181:style:12:6: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?

(shellcheck)